src/com/hotmail/kalebmarc/textfighter/main/Game.java

```diff
@@ -134,6 +136,7 @@ public static void start() {
 			Ui.println("7) Use Insta-Health");
 			Ui.println("8) Use POWER");
 			Ui.println("9) Quit Game (Game will automatically be saved)");
+			Ui.println("10) Run From Battle (You will lose any XP earned)");

```

Comment:
I think it would look better if you made "run from battle" option 9, and move "quit game" to option 10 :)